claim/models.py

```diff
@@ -363,3 +386,36 @@ class ClaimService(core_models.VersionedModel, ClaimDetail):
     class Meta:
         managed = False
         db_table = 'tblClaimServices'
+
+
+class ClaimDedRem(models.Model):
```

Comment:
Inherit from VersionedModel ?
(although save_history and delete_history are probably useless on ClaimDedRem)